Dockerfile

```diff
@@ -115,6 +79,4 @@ WORKDIR /etc/httpd
 
 RUN systemctl enable initialize-httpd-auth sssd httpd dbus-api
 
-VOLUME /sys/fs/cgroup
```

Comment:
with this gone, can we now run this/configmap generator on OpenShift 4 ?

Reply:
I'm not sure it's OpenShift 4 as much as the CRI-O container engine. I haven't gotten the chance to test yet, but the idea is that this image *should* work.

Reply:
Thanks, forgot about CRI-O.  One can only hope 🤞 Thanks @carbonin